src/fingerprint.rs

```diff
@@ -10,8 +10,20 @@ const ZIP32_SEED_FP_PERSONALIZATION: &[u8; 16] = b"Zcash_HD_Seed_FP";
 /// The fingerprint for a wallet's seed bytes, as defined in [ZIP 32].
 ///
 /// [ZIP 32]: https://zips.z.cash/zip-0032#seed-fingerprints
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct SeedFingerprint([u8; 32]);
 
+impl ::core::fmt::Debug for SeedFingerprint {
+    fn fmt(&self, f: &mut ::core::fmt::Formatter) -> ::core::fmt::Result {
+        write!(f, "0x")?;
```

Comment:
I'm not sure I want to include an `0x` prefix; that implies this is a big-endian integer, when it is an opaque bytestring. I'm sure we are inconsistent about this in other places.

More importantly, as this is a `Debug` impl, it does need to indicate that this is a `SeedFingerprint`. There are helper methods that can be used here, but if formatting directly, surround the hex with `SeedFingerprint(` and `)`.

Reply:
Fixed.